.github/workflows/drivers.yml

```diff
@@ -532,6 +532,12 @@ jobs:
       uses: ./.github/actions/test-driver
       with:
         junit-name: 'be-tests-mysql-latest-ee'
+    - name: Upload Test Results as artifact
```

Comment:
why is this here? This should be a separate PR with a full explanation. If we have to revert the liquibase changes i'm guessing we would not want to revert this.

Reply:
Sorry, this is a temp commit. I needed to extract the JUnit results from the workflow somehow.